libs/features/mas/web-components/src/variants/ccd-suggested.css.js

```diff
@@ -0,0 +1,28 @@
+export const CSS = `
+:root {
+  --consonant-merch-card-ccd-suggested-width: 304px;
```

Comment:
I suggest to adopt `--merch-card-ccd-*` for ccd cards.
@npeltier thoughts?

Reply:
i was thinking that we do need some rules indeed :) 
we'll soon have to introduce concept of surface anyway, and i was wondering if with that in mind we still do need to put CCD in our names & variables